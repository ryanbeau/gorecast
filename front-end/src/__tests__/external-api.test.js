import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import ExternalApi from "../views/external-api.js";

it("renders correctly", () => {
  const wrapper = shallow(<ExternalApi />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
