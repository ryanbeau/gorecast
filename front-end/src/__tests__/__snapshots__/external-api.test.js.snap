// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <h1>
    External API
  </h1>
  <p>
    Use these buttons to call an external API. The protected API call has an access token in its authorization header that will be validated by the API server.
  </p>
  <div
    aria-label="External API Requests"
    className="btn-group mt-5"
    role="group"
  >
    <button
      className="btn btn-primary"
      onClick={[Function]}
      type="button"
    >
      Get Public Message
    </button>
    <button
      className="btn btn-primary"
      onClick={[Function]}
      type="button"
    >
      Get Protected Message
    </button>
  </div>
</div>
`;
